src/consumers.ts

```diff
@@ -134,8 +134,8 @@ class EmptyResultEngine extends AnalysisConsumer implements DiagnosticProducer
             return [{
                 severity: DiagnosticSeverity.Information,
                 range: get_range(this.context.version),
-                message: `Application dependency ${this.context.name.value}-${this.context.version.value} - analysis is pending`,
-                source: 'Dependency Analytics Plugin [Powered by Snyk]'
+                message: `Application dependency ${this.context.name.value}-${this.context.version.value} - analysis is pending \n\nView Dependency Analytics report [ctrl/cmd + shift + D]`,
+                source: '\n\n Dependency Analytics plugin [Powered by Snyk]'
```

Comment:
Why `\n\n` is added in the front? Can't be added at the end of `message` property?

Reply:
@arajkumar 
I have tried adding `\n\n` at the end of `message` property but it adding extra space in the begging of new line (screenshot attached.)
![Screenshot from 2020-06-24 16-24-54](https://user-images.githubusercontent.com/42376860/85543667-e5052e80-b637-11ea-87e7-d7558770f8ac.png)

Reply:
@arajkumar PTAL